app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -28,7 +28,7 @@ def import_vm(source_vm_id, target_params)
       :cluster_id        => EmsCluster.find(target_params[:cluster_id]).uid_ems,
       :storage_domain_id => Storage.find(target_params[:storage_id]).ems_ref_obj.split('/').last,
       :sparse            => target_params[:sparse],
-      :drivers_iso       => target_params[:drivers_iso]
+      :drivers_iso       => target_params[:drivers_iso] != '' ? target_params[:drivers_iso] : nil
```

Comment:
we could here there `!target_params[:drivers_iso].to_s.empty ? target_params[:drivers_iso] : nil`